include/common/parallel_execution_native.h

```diff
@@ -42,6 +42,12 @@ struct parallel_execution_native {
   bool ordering = true;
   int num_threads = 4;
   queue_mode lockfree = queue_mode::blocking;
+  int queue_size = 100;
```

Comment:
Because set a queue size is not a requirement. The users may want to use the patterns without defining a size for the queues. In this sense, I think that a size of 100 elements may work well in many cases.